synapse/media/_base.py

```diff
@@ -83,6 +83,53 @@
     "audio/x-flac",
 ]
 
+# Default timeout_ms for download and thumbnail requests
+DEFAULT_MAX_TIMEOUT_MS = 20_000
+
+# Maximum allowed timeout_ms for download and thumbnail requests
+MAXIMUM_ALLOWED_MAX_TIMEOUT_MS = 60_000
+
+
+def parse_media_id(request: Request) -> Tuple[str, str, Optional[str]]:
```

Comment:
This coming back seems like a bad rebase?

Reply:
I don't see any mention of this being a bad change, but this PR has so many comments...

Reply:
I think https://github.com/matrix-org/synapse/commit/b3126a95f4788834d450befe733106282ff35136 will fix it.